src/apps/interactions/controllers/details.js

```diff
@@ -12,6 +12,8 @@ function renderDetailsPage(req, res, next) {
       interaction,
       isComplete
     )
+    const referral = interaction.company_referral
+    referral && (referral.companyId = interaction.company.id)
```

Comment:
I would suggest using a simple if as it's easier to understand.
```suggestion
	    if (referral) {
	    	    referral.companyId = interaction.company.id
	    }
```